src/main/kotlin/uk/gov/justice/digital/hmpps/hmppstier/client/CommunityApiClient.kt

```diff
@@ -15,16 +14,15 @@ import java.time.LocalDate
 class CommunityApiClient(@Qualifier("communityWebClientAppScope") private val webClient: WebClient) {
 
   fun getRegistrations(crn: String): Pair<List<Registration>, List<Registration>> =
-    getRegistrationsCall(crn).sortedByDescending { it.startDate }
-      .partition { ComplexityFactor.from(it.type.code) == IOM_NOMINAL }
+    getRegistrationsCall(crn).partition { it.type.code == IOM_NOMINAL.registerCode }
```

Comment:
The SortedBy is a nonsense that we put in before we understood the requirements.